app/controllers/concerns/serializable.rb

```diff
@@ -8,6 +8,8 @@ def paginate(*args)
 
     paginated_collection = paginate_collection(collection, options)
 
+    yield(paginated_collection, options) if block_given?
```

Comment:
can you add a test for this? or maybe at the controller level that could cover this and the code above?

Reply:
This is covered by the request spec for `GET /location_free_spaces`.

Reply:
https://github.com/ministryofjustice/hmpps-book-secure-move-api/blob/master/spec/requests/api/populations_controller_index_spec.rb#L38

Reply:
https://github.com/ministryofjustice/hmpps-book-secure-move-api/blob/master/spec/requests/mock/pagination_controller_spec.rb